src/script/index.js

```
@@ -1 +1,143 @@
+     * Closes the sidebar navigation, removes focus trap, and restores focus to the last focused element.
+     */
+    const closeSidebar = () => {
+        menuButton.classList.remove('active');
+        menuButton.setAttribute('aria-expanded', false);
+        body.style.overflow = 'auto';
+        const media = window.matchMedia(`(width <= ${mobileMaxBreakpoint})`);
+        document.removeEventListener('click', handleOutsideClick);
+        if (media.matches) {
+            mobileNav.classList.remove('active');
+            document.removeEventListener('keydown', mobileNav._keyHandler);
+            delete mobileNav._keyHandler;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)